tests/publiccloud/slem_basic.pm

```diff
@@ -114,6 +116,26 @@ sub run {
     $instance->run_ssh_command(cmd => 'sudo sestatus | grep enabled');
     $instance->run_ssh_command(cmd => 'sudo dmesg');
     $instance->run_ssh_command(cmd => 'sudo journalctl -p err');
+
+    # volume size tests
+    if (get_var('PUBLIC_CLOUD_ROOT_DISK_SIZE')) {
+        my $size = get_var('PUBLIC_CLOUD_ROOT_DISK_SIZE');
+        my $out = $instance->ssh_script_output(cmd => 'lsblk -J --tree -o SIZE,TYPE');
+        record_info("LSBLK", "$out");
+        my $disks = j $out;
+        $disks = $disks->{blockdevices};
+        my $count_size = 0;
+        for (@$disks) {
+            if (($_->{size} eq ($size . 'G')) && $_->{type} eq 'disk') {
+                $count_size = sum(map { $_->{size} if (chop $_->{size} eq 'G') } @{$_->{children}});
+            }
+        }
+        if (($count_size >= ($size - 3)) && ($count_size <= $size)) {
```

Comment:
Pls, why this `-3` ?

Reply:
safe value for Boot partition reserve + 2Gig reserved space in some configurations